### PR TITLE
Add tesseract.js w/ git auto-update

### PR DESCRIPTION
**Pull request by x09326**
Pull request for issue: #9279 
Related issue(s): # #
@kennynaoh @pvnr0082t 
Please help me review this pull request, thanks.
Because git auto-update can get more version than npm, so I use git auto-update.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required):https://github.com/naptha/tesseract.js
 * Official website (optional, not the repository):http://tesseract.projectnaptha.com/
 * NPM package url (optional):https://www.npmjs.com/package/tesseract.js
 * GitHub / Bitbucket popularity (required):
   - Count of stars:10546
   - Count of watchers:293
   - Count of forks:466
 * NPM download stats (optional):
   - Downloads in the last day:33
   - Downloads in the last week:200
   - Downloads in the last month:830

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by x09326** [comment]
The name of the main file in version 1.0.0(Tesseract.js) is different from other versions(tesseract.js).
I don't know whether it is okay, please help me solve it, thanks.

**Comment by pvnr0082t** [comment]
@x09326 I think maybe we can leave it alone. I'm not sure. Should we change the filename `Tesseract.js` to `tesseract.js` for v1.0.0? @PeterDaveHello

**Comment by PeterDaveHello** [comment]
Any updates here? Thanks!

**Comment by sufuf3** [comment]
Helped add v1.0.8
v1.0.1 can as the same with https://cdn.rawgit.com/naptha/tesseract.js/v1.0.1/dist/tesseract.js and this file is as the same with GitHub. Also, there is no response from upstream since 22 Nov 2016.
So I think the files of v1.0.1 can get from GitHub.

**Comment by sufuf3** [comment]
@cdnjs/active-contributor @cdnjs/intern3 
Please help review this PR, Thank you.

File: ajax/libs/tesseract.js/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  },
+  "homepage": "http://tesseract.projectnaptha.com/",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/naptha/tesseract.js.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/!(.md)"
```
Comment (maruilian11) [inline ajax/libs/tesseract.js/package.json]: I think `**/!(*.md)` would be better?

Comment (PeterDaveHello) [reply ajax/libs/tesseract.js/package.json]: Looks so.

File: ajax/libs/tesseract.js/package.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "tesseract.js",
+  "filename": "tesseract.min.js",
+  "version": "1.0.10",
+  "description": "Pure Javascript Multilingual OCR",
+  "author": "antimatter15",
```
Comment (maruilian11) [inline ajax/libs/tesseract.js/package.json]: The real name will be better.

Comment (PeterDaveHello) [reply ajax/libs/tesseract.js/package.json]: We don't need real name, and how can you know if it's real name or not?
It's better to use what it displays on GitHub page.

**Comment by PeterDaveHello** [comment]
@sufuf3 can you help solve the issues?

**Comment by sufuf3** [comment]
Yes, I can! 
Working on this.

**Comment by PeterDaveHello** [comment]
@sufuf3 Working on your own PRs first please, there are many waiting for your updates.

**Comment by sufuf3** [comment]
I helped update the pattern.
@cdnjs/intern3 Please help review this PR, thanks.

File: ajax/libs/tesseract.js/package.json (hunk 2 above)
Comment (maruilian11) [reply ajax/libs/tesseract.js/package.json]: https://github.com/antimatter15
Doesn't `Kevin Kwok` sound better?

Comment (maruilian11) [reply ajax/libs/tesseract.js/package.json]: fine

**Comment by PeterDaveHello** [comment]
@sufuf3 I thought you didn't update the author info?

**Pull request merged**